### PR TITLE
Add option for root node name on Import 3D scene window

### LINKED ISSUE #6822: Ability to name root node of imported asset scene after its filename
**Operating system or device - Godot version:**
Any

**Issue description** (what happened, and what was expected):
having to rename the root node of a imported scene can get tedious when dealing with lots of scenes or scenes that get updated over time. This can lead to issues if you have code/nodes that depend on some form of naming convention. I think that having a check box in the import settings to rename the root node to the same as the source file name should help alleviate some of the headache. We usually name the source file in a manner that represents what it contains anyway.

**Steps to reproduce:**

**Link to minimal example project** (optional but very welcome):

### PR DESCRIPTION
**Pull request by volzhs**
Fix #6822

image

### CODE REVIEW

**Comment by volzhs** [comment]
Added option for root node name, not just taking filename as @reduz commented https://github.com/godotengine/godot/issues/6822#issuecomment-253791761

File: tools/editor/io_plugins/editor_scene_import_plugin.cpp [hunk 1]
```diff
@@ -656,6 +658,9 @@ void EditorSceneImportDialog::_choose_file(const String& p_path) {
 
 
 	import_path->set_text(p_path);
+	if (root_node_name->get_text().size()==0){
+		root_node_name->set_text(p_path.get_file().basename());
```
Comment (akien-mga) [inline tools/editor/io_plugins/editor_scene_import_plugin.cpp]: Could you also prefill the LineEdit with the filename, using LineEdit's placeholder text feature?

Comment (volzhs) [inline tools/editor/io_plugins/editor_scene_import_plugin.cpp]: No problem.

Comment (volzhs) [inline tools/editor/io_plugins/editor_scene_import_plugin.cpp]: Done. :)

File: tools/editor/io_plugins/editor_scene_import_plugin.cpp [hunk 2]
```diff
@@ -656,6 +658,9 @@ void EditorSceneImportDialog::_choose_file(const String& p_path) {
 
 
 	import_path->set_text(p_path);
+	if (root_node_name->get_text().size()==0){
+		root_node_name->set_text(import_path->get_text().get_file().basename());
```
Comment (akien-mga) [inline tools/editor/io_plugins/editor_scene_import_plugin.cpp]: I was actually thinking about `LineEdit::set_placeholder`: https://github.com/godotengine/godot/blob/master/scene/gui/line_edit.cpp#L934-L938

It shows a placeholder text (with lower alpha value) that you can overwrite by typing, that fitted well with your previous implementation. But pre-filling "for real" the text is fine too IMO, I don't have a strong opinion about which is best.

**Pull request merged**